### PR TITLE
add organization & expected_media_types for registries to schema

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: atomic_reactor/schemas/config.json [hunk 1]
```diff
@@ -451,6 +468,10 @@
                 "description": "Registry url",
                 "type": "string"
             },
+            "organization": {
```
Comment (twaugh) [inline atomic_reactor/schemas/config.json]: I wonder if we want a "definition" for a registry, so we can reference it from both of these places.

Comment (lcarva) [reply atomic_reactor/schemas/config.json]: Probably a good idea.

Comment (rcerven) [reply atomic_reactor/schemas/config.json]: ack will do

**Comment by rcerven** [comment]
added organization to definitions

**Comment by rcerven** [comment]
rn updated
- Added organization & expected_media_types for registries to schema

**Pull request merged**